web-registry/src/App.tsx

```diff
@@ -182,6 +183,10 @@ const App: React.FC = (): JSX.Element => {
               />
             </Route>
           </Route>
+          <Route
+            path="credit-batch-issuance"
```

Comment:
what about renaming this to something more consistent with ledger API naming? (both the route and the component name)
e.g. `ecocredit/create-batch`
(also turning our current subroute `ecocredits` into `ecocredit` to be aligned with "ecocredit module")

Reply:
hmm, I like aligning with the modules, but in some cases, the plural routes are more user-friendly, IMO. I'll think about this for a bit.

Reply:
Changing to `ecocredits/create-batch` for now. I was worried about breaking other links if I rearrange all the ecocredit stuff. I can make the change if we get more feedback. 
cc @erikalogie